frontend/src/scenes/toolbar-launch/ToolbarLaunch.tsx

```diff
@@ -0,0 +1,76 @@
+import { PageHeader } from 'lib/components/PageHeader'
+import React from 'react'
+import { SceneExport } from 'scenes/sceneTypes'
+import './ToolbarLaunch.scss'
+import { SearchOutlined } from '@ant-design/icons'
+import { Link } from 'lib/components/Link'
+import { urls } from 'scenes/urls'
+import { IconFlag, IconGroupedEvents, IconHeatmap } from 'lib/components/icons'
+import { Col, Row } from 'antd'
+import { AuthorizedUrlsTable } from './AuthorizedUrlsTable'
+
+export const scene: SceneExport = {
+    component: ToolbarLaunch,
+}
+
+function ToolbarLaunch(): JSX.Element {
+    const features: FeatureHighlightProps[] = [
+        {
+            title: 'Heatmaps',
+            caption: 'Understand where your users interact the most.',
+            icon: <IconHeatmap />,
+        },
+        {
+            title: 'Actions',
+            caption: 'Create actions visually from elements in your website.',
+            icon: <IconGroupedEvents />,
+        },
+        {
+            title: 'Feature Flags',
+            caption: 'Toggle feature flags on/off right on your app.',
+            icon: <IconFlag />,
+        },
+        {
+            title: 'Inspect',
+            caption: 'Inspect clickable elements on your website.',
+            icon: <SearchOutlined />,
+        },
+    ]
+
+    return (
+        <div className="toolbar-launch-page">
+            <PageHeader title="Toolbar" caption="The toolbar launches PostHog right in your app or website." />
+
+            <AuthorizedUrlsTable pageKey="toolbar-launch" />
+
+            <div className="footer-caption">
+                Make sure you're using the <Link to={`${urls.projectSettings()}#snippet`}>snippet</Link> or the latest{' '}
```

Comment:
"the snippet" sounds very ambiguous, I think a simple clarification may be just saying "the HTML snippet"